dask_sql/context.py

```diff
@@ -587,69 +592,42 @@ def register_model(
         self.schema[schema_name].models[model_name.lower()] = (model, training_columns)
 
     def set_config(
-        self,
-        config_options: Union[Tuple[str, Any], Dict[str, Any]],
-        schema_name: str = None,
+        self, config_options: Dict[str, Any],
     ):
         """
-        Add configuration options to a schema.
-        A configuration option could be used to set the behavior of certain configurirable operations.
+        Sets sql configuration options to the global dask config
+        A configuration option could be used to set the behavior of certain sql operations.
 
-        Eg: `dask.groupby.agg.split_out` can be used to split the output of a groupby agrregation to multiple partitions.
+        Eg: `sql.groupby.split_out` can be used to split the output of a groupby agrregation to multiple partitions.
 
         Args:
-            config_options (:obj:`Tuple[str,val]` or :obj:`Dict[str,val]`): config_option and value to set
-            schema_name (:obj:`str`): Optionally select schema for setting configs
+            config_options (:obj:`Dict[str,val]`): key, value paris of config_option and value to set.
 
         Example:
             .. code-block:: python
 
                 from dask_sql import Context
 
                 c = Context()
-                c.set_config(("dask.groupby.aggregate.split_out", 1))
+                c.set_config({"sql.groupby.split_out", 1})
                 c.set_config(
                     {
-                        "dask.groupby.aggregate.split_out": 2,
-                        "dask.groupby.aggregate.split_every": 4,
+                        "sql.groupby.split_every": 2,
```

Comment:
I don't have a strong preference for either. It was originally called `groupby.aggregate.split_out/every` since the `split_out` arg is technically passed to the aggregate api call not the groupby call itself. 
In this PR I renamed it to just `groupby.split_out/every` since it has a lower level of nesting and I think conveys the same idea for what this option is trying to do.